learning-development/accessibility.qmd

```diff
@@ -39,7 +44,7 @@ For more advice about colour in charts and visualisations specifically, includin
 
 ## Free tools for reviewing web pages
 
-- [axe DevTools](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?pli=1), free chrome extension and paid versions, we recommend this (and GDS tend to recommend too)
+- [axe DevTools](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?pli=1), free Google Chrome extension and paid versions, we recommend this (and GDS tend to recommend too)
```

Comment:
Sorry - missed this the first time but I'd consider spelling out GDS in full if we haven't already as I'm not sure everyone is familiar with them